src/main/drivers/dshot_bitbang_decode.h

```diff
@@ -18,14 +18,14 @@
  * If not, see <http://www.gnu.org/licenses/>.
  */
 
-
+#include "platform.h"
 
 #if defined(USE_DSHOT) && defined(USE_DSHOT_TELEMETRY)
 
-
-
-uint32_t decode_bb(uint16_t buffer[], uint32_t count, uint32_t mask);
+#if defined(STM32F4) || defined(AT32F4)
```

Comment:
Shouldn’t this use `#ifdef USE_DSHOT_BITBAND`?

Reply:
Thanks. Good catch. Fixed.